### ISSUE TITLE
Cannot install package, broken dependency "cosmiconfig#3.0"

### ISSUE THREAD

**Issue by emmenko** [cjoudrey/graphql-schema-linter#47]
```bash
$ yarn
yarn install v0.27.5
[1/4] Resolving packages...
error Couldn't find match for "3.0" in "master,revert-2.2.0,10/head,13/head,14/head,15/head,16/head,17/head,19/head,22/head,23/head,24/head,24/merge,27/head,29/head,29/merge,31/head,33/head,34/head,34/merge,36/head,37/head,37/merge,42/head,42/merge,44/head,44/merge,47/head,51/head,52/head,52/merge,54/head,55/head,60/head,60/merge,61/head,61/merge,64/head,66/head,67/head,68/head,70/head,73/head,74/head,75/head,76/head,78/head,0.1.0,0.2.0,1.0.0,1.0.1,1.0.2,1.1.0,2.0.0,2.0.1,2.0.2,2.1.0,2.1.1,2.1.2,2.1.3,2.2.0,2.2.1,2.2.2" for "https://github.com/davidtheclark/cosmiconfig.git".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

Refs: https://github.com/cjoudrey/graphql-schema-linter/pull/26#discussion_r132853023

Apparently the `3.0` branch has been [merged 2 days ago and deleted](https://github.com/davidtheclark/cosmiconfig/pull/78). Please use a normal version thanks.

**Comment by mellocello2003** [comment]
This is a major issue since it affects released versions.  Can we get an update on this?  Thanks!

**Comment by cjoudrey** [comment]
Sorry for the radio silence only saw this last night. Merged #48 and released 0.0.17 which should fix this. Thanks for reporting the issue. 😀 

I need to check why I didn't get a GitHub notification when this issue was created. 🤔
